internal/pkg/cli/flag.go

```diff
@@ -156,6 +157,8 @@ Mutually exclusive with -%s, --%s.`, imageFlagShort, imageFlag)
 
 	clusterFlagDescription = fmt.Sprintf(`Optional. The short name or full ARN of the cluster to run the task in. 
 Cannot be specified with '%s', '%s' or '%s'.`, appFlag, envFlag, taskDefaultFlag)
+	acknowledgeSecretsAccessDescription = fmt.Sprintf(`Optional. If provided, does not ask grant permission for secrets. 
+Cannot be specified without '%s.`, acknowledgeSecretsAccessFlag)
```

Comment:
oops, I meant to say Cann't use without secrets flag. Secrets flag has to be there right?

Reply:
I am not sure if we should add this to the description since we don't really error out or anything if `--acknowledge-secrets-access` is specified while `--secrets` is not.

Instead, we could probably say something like `"This flag is useful only when --secrets is specified"`, or like `"Usage of this flag is only checked when --secrets is specified"`